develop/toolkit/blockchain/spawn-testing-chain.md

```diff
@@ -1,4 +1,1213 @@
 ---
 title: Spawn Chains for Testing with Zombienet
-description: TODO
----
\ No newline at end of file
+description: Learn how to deploy, configure, and test Polkadot SDK-based blockchain networks using Zombienet with Kubernetes, Podman, or local setups.
+---
+
+# Spawn Chains for Testing with Zombienet
+
+## Introduction
+
+Zombienet is a versatile testing framework tailored for Polkadot SDK-based blockchain networks. This guide consolidates essential tasks for deploying and testing ephemeral blockchain environments across various platforms, including Kubernetes, Podman, and native setups. With its simple CLI, developers can spawn relay chains, parachains, and run comprehensive tests to validate network performance. 
+
+From installation to advanced testing scenarios, you’ll explore how Zombienet leverages metrics, logs, and custom scripts to ensure robust blockchain development. Whether you're new to blockchain testing or refining your setup, this guide offers the tools and insights needed for efficient testing.
+
+??? interface "Additional support resources"
+    [Parity Technologies](https://www.parity.io/){target=_blank} has designed and developed this framework, now maintained by the Zombienet team. 
+
+    For further support and information, refer to the following contact points:
+
+    - [Zombienet repository](https://github.com/paritytech/zombienet){target=_blank}
+    - [Element public channel](https://matrix.to/#/!FWyuEyNvIFygLnWNMh:parity.io?via=parity.io&via=matrix.org&via=web3.foundation){target=_blank}
+
+## Install Zombienet
+
+Zombienet releases are available on the [Zombienet repository](https://github.com/paritytech/zombienet){target=_blank}.
+
+To install Zombienet, there are multiple options available, depending on the user's preferences and the environment where it will be used. The following section will guide you through the installation process for each of the available options.
```

Comment:
Can you double check this line?

For some reason in my .md file of this, it shows as 
`Toinstall Zombienet, there are multiple options available, depending on the user's preferences and the environment where it will be used. The following section will guide you through the installation process for each of the available options.`

 but correctly renders on Github. Not sure why, if you don't have the same don't worry

Reply:
Looks fine when I spin it up.